src/Provider/GoogleMapsPlaces/Model/GooglePlace.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\GoogleMapsPlaces\Model;
+
+use Geocoder\Model\Address;
+
+/**
+ * @author atymic <atymicq@gmail.com>
+ */
+final class GooglePlace extends Address
+{
+    /**
+     * @var string|null
+     */
+    private $id;
+
+    /**
+     * @var string|null
+     */
+    private $name;
+
+    /**
+     * @var array
+     */
+    private $resultType = [];
```

Comment:
Why call it `resultType` instead of just `type` ?
I think `type` would be more clear.

Reply:
Updated.